### PR TITLE
 Adds documentation and caveats about image formats system within rich text

### LINKED ISSUE #1600: Documentation required regarding Rich Text image alignment classes
See: https://github.com/torchbox/wagtail/issues/1599

A "Note" is required somewhere in the docs to point out that while Wagtail ships with 3 alignments for images embedded within rich text, no CSS is provided or injected anywhere to make the alignments display the same on the front end. This is clearly confusing, but just needs clarifying.

### PR DESCRIPTION
**Pull request by davecranwell**
Fixes #1600.

### CODE REVIEW

File: docs/topics/images/using_in_templates.rst [hunk 1]
```diff
@@ -170,3 +170,32 @@ You can also use the ``attrs`` property as a shorthand to output the attributes
...
+
+
+Images embedded in rich text
+----------------------------
+
+The information above relates to images defined via image-specific fields in your model, but images can also be embedded arbitrarily in Rich Text fields by the editor (see :ref:`rich-text`).
+
+Images embedded in Rich Text fields can't be controlled by the template developer as easily. There are no image objects to work with, so the ``{% image %}`` can't be used. Instead editors can choose from one of a number of image "Formats" at the point of inserting images into their text.
+
+Wagtail comes with three pre-defined image formats, but more can be defined in python by the developer. These formats are:
```
Comment (tomdyson) [inline docs/topics/images/using_in_templates.rst]: python -> Python

File: docs/topics/images/using_in_templates.rst [hunk 2]
```diff
@@ -170,3 +170,32 @@ You can also use the ``attrs`` property as a shorthand to output the attributes
...
+
+    ``Right-aligned``
+        Creates an image tag with the filter ``width-500`` and given the CSS class ``right``
+
+.. Note::
+
+    The CSS classes added to images do **not** come with any accompanying stylesheets, or inline styles. e.g the ``left`` class will do nothing, by default. The developer is expected to add these classes to their front end CSS files, to define what exactly ``left``, ``right`` or ``full-width`` means *to them*.
+
+For more information about image formats, including creating your own, see :ref:`rich_text_image_formats`
+
```
Comment (kaedroho) [inline docs/topics/images/using_in_templates.rst]: :scissors:

Comment (davecranwell) [inline docs/topics/images/using_in_templates.rst]: Explain?

Comment (kaedroho) [inline docs/topics/images/using_in_templates.rst]: two blank lines at the bottom

**Comment by kaedroho** [comment]
Merged in: 12fc3e6601dcdc35764f9104b3f93289f90095bb

**Pull request closed without merging**